boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_shared.dts

```diff
@@ -0,0 +1,133 @@
+/ {
+leds:	leds {
+		compatible = "gpio-leds";
```

Comment:
Does the main RGB LED (center) have PWM capability? If so, it would be nice to add the pwmleds & aliases.

Reply:
We can do that, but I would like to keep it separate from this PR. In the current stage, we want to move this board and not add or remove any features.